target/scripts/start-mailserver.sh

```diff
@@ -31,142 +31,18 @@ source /usr/local/bin/daemons-stack.sh
 # ------------------------------------------------------------
 # ? << Sourcing helpers & stacks
 # --
-# ? >> Setup Supervisor & DNS names
+# ? >> Early setup & environment variables setup
 # ------------------------------------------------------------
 
-# Setup supervisord as early as possible
-declare -A VARS
-VARS[SUPERVISOR_LOGLEVEL]="${SUPERVISOR_LOGLEVEL:=warn}"
+# shellcheck source=./helpers/variables.sh
+source /usr/local/bin/helpers/variables.sh
 
-_setup_supervisor
-_obtain_hostname_and_domainname
+_environment_variables_early_setup
+_environment_variables_backwards_compatibility
+_environment_variables_general_setup
 
 # ------------------------------------------------------------
-# ? << Setup Supervisor & DNS names
-# --
-# ? >> Setup of default and global values / variables
-# ------------------------------------------------------------
-
-# shellcheck disable=SC2034
-declare -a FUNCS_SETUP FUNCS_FIX FUNCS_CHECK FUNCS_MISC DAEMONS_START
-
-# These variables must be defined first; They are used as default values for other variables.
-VARS[POSTMASTER_ADDRESS]="${POSTMASTER_ADDRESS:=postmaster@${DOMAINNAME}}"
-VARS[REPORT_RECIPIENT]="${REPORT_RECIPIENT:=${POSTMASTER_ADDRESS}}"
-VARS[REPORT_SENDER]="${REPORT_SENDER:=mailserver-report@${HOSTNAME}}"
-
-VARS[AMAVIS_LOGLEVEL]="${AMAVIS_LOGLEVEL:=0}"
-VARS[CLAMAV_MESSAGE_SIZE_LIMIT]="${CLAMAV_MESSAGE_SIZE_LIMIT:=25M}" # 25 MB
-VARS[DEFAULT_RELAY_HOST]="${DEFAULT_RELAY_HOST:=}"
-VARS[DOVECOT_INET_PROTOCOLS]="${DOVECOT_INET_PROTOCOLS:=all}"
-VARS[DOVECOT_MAILBOX_FORMAT]="${DOVECOT_MAILBOX_FORMAT:=maildir}"
-VARS[DOVECOT_TLS]="${DOVECOT_TLS:=no}"
-VARS[ENABLE_AMAVIS]="${ENABLE_AMAVIS:=1}"
-VARS[ENABLE_CLAMAV]="${ENABLE_CLAMAV:=0}"
-VARS[ENABLE_DNSBL]="${ENABLE_DNSBL:=0}"
-VARS[ENABLE_FAIL2BAN]="${ENABLE_FAIL2BAN:=0}"
-VARS[ENABLE_FETCHMAIL]="${ENABLE_FETCHMAIL:=0}"
-VARS[ENABLE_LDAP]="${ENABLE_LDAP:=0}"
-VARS[ENABLE_MANAGESIEVE]="${ENABLE_MANAGESIEVE:=0}"
-VARS[ENABLE_POP3]="${ENABLE_POP3:=0}"
-VARS[ENABLE_POSTGREY]="${ENABLE_POSTGREY:=0}"
-VARS[ENABLE_QUOTAS]="${ENABLE_QUOTAS:=1}"
-VARS[ENABLE_SASLAUTHD]="${ENABLE_SASLAUTHD:=0}"
-VARS[ENABLE_SPAMASSASSIN]="${ENABLE_SPAMASSASSIN:=0}"
-VARS[ENABLE_SPAMASSASSIN_KAM]="${ENABLE_SPAMASSASSIN_KAM:=0}"
-VARS[ENABLE_SRS]="${ENABLE_SRS:=0}"
-VARS[ENABLE_UPDATE_CHECK]="${ENABLE_UPDATE_CHECK:=1}"
-VARS[FAIL2BAN_BLOCKTYPE]="${FAIL2BAN_BLOCKTYPE:=drop}"
-VARS[FETCHMAIL_PARALLEL]="${FETCHMAIL_PARALLEL:=0}"
-VARS[FETCHMAIL_POLL]="${FETCHMAIL_POLL:=300}"
-VARS[LDAP_START_TLS]="${LDAP_START_TLS:=no}"
```

Comment:
This was removed because I thought this was unused, but I guess this was an error on my side. The relevant commits are:

1. 8fc4ea3e0da4e23fc42bcead7f32d362b9271dbf removal in error
2. 7dbe737a69f6057e630b589d10e698317e0c0f00 adding it again

Sorry, issue on my side.